website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -48,17 +48,21 @@ Interacting with notification configurations requires admin access to the releva
 
 Notifications are sent as the run progresses, and can be triggered on one or more types of state transition. These are specified in the `triggers` array attribute. Available triggers are:
 
-| Display Name    | Value                   | Description                                                                                              |
-| --------------- | ----------------------- | -------------------------------------------------------------------------------------------------------- |
-| Created         | `"run:created"`         | When a run is created and enters the "Pending" state.                                                    |
-| Planning        | `"run:planning"`        | When a run acquires the lock and starts to execute.                                                      |
-| Needs Attention | `"run:needs_attention"` | Human decision required. When a plan has changes and is not auto-applied, or requires a policy override. |
-| Applying        | `"run:applying"`        | When a run begins the apply stage, after a plan is confirmed or auto-applied.                            |
-| Completed       | `"run:completed"`       | When the run has completed on a happy path and can't go any further.                                     |
-| Errored         | `"run:errored"`         | When the run has terminated early due to error or cancellation.                                          |
+| Display Name       | Value                   | Description                                                                                              |
+| ------------------ | ----------------------- | -------------------------------------------------------------------------------------------------------- |
+| Created            | `"run:created"`         | When a run is created and enters the "Pending" state.                                                    |
+| Planning           | `"run:planning"`        | When a run acquires the lock and starts to execute.                                                      |
+| Needs Attention    | `"run:needs_attention"` | Human decision required. When a plan has changes and is not auto-applied, or requires a policy override. |
+| Applying           | `"run:applying"`        | When a run begins the apply stage, after a plan is confirmed or auto-applied.                            |
+| Completed          | `"run:completed"`       | When the run has completed on a happy path and can't go any further.                                     |
+| Errored            | `"run:errored"`         | When the run has terminated early due to error or cancellation.                                          |
```

Comment:
```suggestion
| Created            | `"run:created"`         | A run is created and enters the [Pending stage](/cloud-docs/run/states#1-the-pending-stage)                                                    |
| Planning           | `"run:planning"`        | A run acquires the lock and starts to execute.                                                      |
| Needs Attention    | `"run:needs_attention"` | A plan has changes and Terraform requires user input to continue. This may include approving the plan or a [policy override](/cloud-docs/run/states#5-the-policy-check-stage). |
| Applying           | `"run:applying"`        | A run enters the [Apply stage](/cloud-docs/run/states#6-the-apply-stage), where Terraform makes the infrastructure changes described in the plan.                            |
| Completed          | `"run:completed"`       | A run has completed successfully.                                     |
| Errored            | `"run:errored"`         | A run has terminated early due to error or cancellation.                                          |
```